netutils/netlib/netlib_ipv6route.c

```diff
@@ -131,9 +131,12 @@ ssize_t netlib_read_ipv6route(FILE *stream,
       return 0;
     }
 
-  /* The first line of the group should consist of a number index */
+  /* The first non-space char of the first line of the group should consist of a number index */
+  int idx = 0;
+  while (isspace(line[idx]))
+    idx++;
```

Comment:
```suggestion
  while (isspace(line[idx]))
    {
      idx++;
    }
```
but anyway I do not see how this could should help if below we still check `line[0]` and not `line[idx]`.
Also line `addr = &line[ADDR_OFFSET];` know nothing about `idx`.

To me this change looks quite suspicious.

Reply:
The second commit in the PR fixes the format comments and a couple of automated warnings.  I did forget to update `line[0] `to `line [idx]` when creating the PR from a clean fork.  This is corrected in the second PR, thanks for catching it.
As far as `addr = &line[ADDR_OFFSET];` not knowing about idx, I do not see a problem with that.  That line just wants to make sure there is a NULL teminator at the max PROCFS line length.  Idx is only there to avoid a failed check of ` if (line[0] < '0' || line[0] > '9')` as was the case in the original code.

Reply:
Please squash the change into one patch, @mlondono74